src/EnergyPlus/HVACSizingSimulationManager.cc

```diff
@@ -184,7 +184,7 @@ void HVACSizingSimulationManager::ProcessCoincidentPlantSizeAdjustments(EnergyPl
     }
 
     // as more sizing adjustments are added this will need to change to consider all not just plant coincident
-    state.dataGlobal->FinalSizingHVACSizingSimIteration = plantCoinAnalyRequestsAnotherIteration;
+    //    state.dataGlobal->FinalSizingHVACSizingSimIteration = plantCoinAnalyRequestsAnotherIteration;
```

Comment:
Commented out this unused line, leaving it in to provide additional context to the line above for future expansion.